### PR TITLE
SES: Implement SendBulkTemplatedEmail

### PR DESCRIPTION
**Pull request by deepaksrgm2010**
Feature: Implemented SendBulkTemplatedEmail API feature from SES.

This is [#5360](https://github.com/spulec/moto/issues/5360)

### CODE REVIEW

File: moto/ses/responses.py [hunk 1]
```diff
@@ -94,6 +94,35 @@ def send_templated_email(self):
...
+            destination = {"ToAddresses": [], "CcAddresses": [], "BccAddresses": []}
+            for dest_type in destination:
+                # consume up to 51 to allow exception
+                for j in range(1, 52):
+                    field = "Destinations.member.%s.Destination.%s.member.%s" % (i, dest_type, j)
+                    address = self.querystring.get(field)
+                    if address is None:
+                        break
+                    destination[dest_type].append(address[0])
+            destinations.append({"Destination": destination})
```
Comment (bblommers) [inline moto/ses/responses.py:116]: Can we retrieve these parameters using one of the existing helper methods, such as `self._get_params()` or `self._get_multi_param_dict`?

File: tests/test_ses/test_ses_boto3.py [hunk 2]
```diff
@@ -174,6 +174,68 @@ def test_send_email_invalid_address():
...
+        }
+    )
+
+    conn.send_bulk_templated_email(**kwargs)
+
+    too_many_destinations = list({"Destination": {"ToAddresses": ["to%s@example.com" %i ], "CcAddresses": [],
+                                                  "BccAddresses": [] }} for i in range(51))
+    conn.send_bulk_templated_email.when.called_with(
+        **dict(kwargs, Destinations=too_many_destinations)
+    ).should.throw(ClientError)
```
Comment (bblommers) [inline tests/test_ses/test_ses_boto3.py:225]: We should verify that the correct error was thrown, like you did above:
```
with pytest.raises() as ex:
    ....
ex... should.equal(...)
```

File: tests/test_ses/test_ses_boto3.py [hunk 3]
```diff
@@ -174,6 +174,68 @@ def test_send_email_invalid_address():
...
+
+    too_many_destinations = list("to%s@example.com" % i for i in range(51))
+
+    conn.send_bulk_templated_email.when.called_with(
+         **dict(kwargs, Destinations=[{"Destination": {"ToAddresses": too_many_destinations,
+                                                       "CcAddresses": [], "BccAddresses": []}}])
+    ).should.throw(ClientError)
```
Comment (bblommers) [inline tests/test_ses/test_ses_boto3.py:232]: Same

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @deepaksrgm2010, thanks for the PR! Please see my comments.

Note that the linter is currently failing as well, that should also be fixed

File: moto/ses/responses.py (hunk 1 above)
Comment (deepaksrgm2010) [reply moto/ses/responses.py:116]: @bblommers
-  I tried using self._get_params(). There is `index out of bound` error when its used decode query string whose memebers are out of order (I know becuase my tests failed when i used them. but only on specific cases).
Note: I plan have look at this helper method later and fix it if possible
- Rest of the methods in this module uses the same convention I have. So for now can we keep it this way.

**Comment by deepaksrgm2010** [comment]
Apologies for the lint error. Some error in my lint set up. Corrected it.

**Review by bblommers — APPROVED** [review]
LGTM - thanks @deepaksrgm2010!

**Pull request merged**